### ISSUE TITLE
FUN.9 Mode Change?

### ISSUE THREAD

**Issue by mluckcuck** [foselab/abz2024_casestudy_MLV#5]
FUN.9 says "Once the self-test has been completed successfully and configurations have been loaded properly the system shall start monitoring and reporting health parameters." is this supposed to indicate a mode change to 'Ventilation Mode' (as per Figure 2.1)?

**Comment by garganti** [comment]
Yes, this means that the mode changes to StandBy Mode. The ventilator is powered, it is displaying the ventilation parameters, but no one is connected yet. The change to ventilation mode happens when the operator starts the ventilation (likely with a patient connected to the machine).  We will clarify this in a new version of the document.

**Comment by mluckcuck** [comment]
Ah yes, sorry Fig 2.1 says Standby Mode is next. I think "monitoring and reporting..." made it sound like that was Ventilation Mode.

So, FUN.9 triggers the transition to Standby Mode but so does FUN10.4. Is there a reason for this repetition?

**Comment by silviabonfanti** [comment]
No, no reason
